<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>managetasks</class>
 <widget class="QDialog" name="managetasks">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>819</width>
    <height>522</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="m_clear">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>270</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="m_delete">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>180</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="m_OK">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>440</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_Cancel">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>440</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>50</y>
     <width>471</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
